README.md

```
@@ -0,0 +1,47 @@
+Individual reports are deleted after combining.
+
+## Test Workers
+
+Test workers must:
+1. Accept parameters via `TEST_*` environment variables
+2. Generate a JUnit XML report
+3. Write the report to `/reports/{test_name}_report.xml`
+
+The `TEST_NAME` environment variable contains the test name from the config.
+
+Tests can be found [HERE](https://github.com/vliz-be-opsci/grmp-test-implementations)
```

markdownlint-cli2 (0.20.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)